app/src/main/java/ayds/lisboa/songinfo/home/model/repository/local/spotify/sqldb/CursorToSpotifySongMapper.kt

```diff
@@ -21,6 +24,7 @@ internal class CursorToSpotifySongMapperImpl : CursorToSpotifySongMapper {
                       artistName = getString(getColumnIndexOrThrow(ARTIST_COLUMN)),
                       albumName = getString(getColumnIndexOrThrow(ALBUM_COLUMN)),
                       releaseDate = getString(getColumnIndexOrThrow(RELEASE_DATE_COLUMN)),
+                      releaseDatePrecision = getSongReleaseDatePrecision(),
```

Comment:
Por consistencia y nivel de abstracción, en este caso mejor hacer directamente: 
```suggestion
                      releaseDatePrecision = getInt(getColumnIndexOrThrow(RELEASE_DATE_PRECISION_COLUMN)),
```